service/controller/v2/arm_templates/kubernetes_load_balancer_setup.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "initialProvisioning": {
+      "type": "string",
+      "defaultValue": "Yes",
+      "metadata": {
+        "description": "Whether the deployment is provisioned the very first time."
+      }
+    }
+  },
+  "resources": [
+    {
+      "apiVersion": "2017-10-01",
+      "type": "Microsoft.Network/publicIPAddresses",
+      "name": "dummy-pip",
+      "location": "[resourceGroup().location]",
+      "properties": {}
+    },
+    {
+      "apiVersion": "2017-10-01",
+      "name": "kubernetes",
+      "type": "Microsoft.Network/loadBalancers",
+      "condition": "[equals(parameters('initialProvisioning'), 'Yes')]",
```

Comment:
do not LB reconcile this after initial installation as it will be managed by kubernetes